registration-processor/pre-processor/registration-processor-securezone-notification-stage/src/main/java/io/mosip/registration/processor/securezone/notification/stage/SecurezoneNotificationStage.java

```diff
@@ -157,23 +158,68 @@ public void processURL(RoutingContext ctx) {
 		regProcLogger.debug(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.REGISTRATIONID.toString(), "",
 				"SecurezoneNotificationStage::processURL()::entry");
 
+		MessageDTO request = new MessageDTO();
+		JsonObject obj = ctx.getBodyAsJson();
+
+		request.setMessageBusAddress(MessageBusAddress.SECUREZONE_NOTIFICATION_IN);
+		request.setInternalError(Boolean.FALSE);
+		request.setRid(obj.getString("rid"));
+		request.setReg_type(obj.getString("reg_type"));
+		request.setIsValid(obj.getBoolean("isValid"));
+		request.setSource(obj.getString("source"));
+		request.setIteration(obj.getInteger("iteration"));
+		request.setWorkflowInstanceId(obj.getString("workflowInstanceId"));
+
+		MessageDTO messageDTO = process(request);
+
+		if (messageDTO.getInternalError()) {
+			ctx.fail(new BaseUncheckedException("Could not process request. Please check logs for more details."));
+		}else if (messageDTO.getIsValid()) {
+			sendMessage(messageDTO);
+			this.setResponse(ctx,
+					"Packet with registrationId '" + messageDTO.getRid() + "' has been forwarded to next stage");
+			regProcLogger.info(obj.getString("rid"),
+					"Packet with registrationId '" + messageDTO.getRid() + "' has been forwarded to next stage",
+					null, null);
+		} else {
+			this.setResponse(ctx, "Packet with registrationId '" + obj.getString("rid")
+					+ "' has not been uploaded to file System");
+			regProcLogger.info(obj.getString("rid"),
+					"Packet with registrationId '" + messageDTO.getRid() + "' has not been uploaded to file System",
+					null, null);
+		}
+	}
+
+	/**
+	 * This is for failure handler
+	 *
+	 * @param routingContext
+	 */
+	private void failure(RoutingContext routingContext) {
+		this.setResponse(routingContext, routingContext.failure().getMessage());
+	}
+
+	/**
+	 * sends messageDTO to camel bridge.
+	 *
+	 * @param messageDTO the message DTO
+	 */
+	public void sendMessage(MessageDTO messageDTO) {
+		if (routingEnabled)
```

Comment:
consumeAndSend will obey this routingEnabled flag ? If not how are we taking care of that ? 
Can we use the "mosip.regproc.stage-common.bus-out-halt-addresses" config and remove the "securezone.routing.enabled" altogether since it will be confusing ?